AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -3359,28 +3359,18 @@ function Test-Port445Connectivity
     [CmdletBinding()]
 
     param (
-        [Parameter(Mandatory=$True, Position=0, HelpMessage="Storage account name")]
-        [string]$StorageAccountName,
-
-        [Parameter(Mandatory=$True, Position=1, HelpMessage="Resource group name")]
-        [string]$ResourceGroupName
+        [Parameter(Mandatory=$True, Position=0, HelpMessage="Account FileEndPoint")]
+        [string]$StorageAccountFileEndPoint

```

Comment:
This is a good change. Though if you change this, make sure you also change the call in `Debug-AzStorageAccountADDSAuth`